sa/_db-next/migrations/20170731115209_AddOrders.sql

```diff
@@ -0,0 +1,25 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE orders (
+       id bigint(20) NOT NULL AUTO_INCREMENT,
+       registrationID bigint(20) NOT NULL,
+       expires datetime NOT NULL,
+       csr mediumblob NOT NULL,
+       error blob DEFAULT NULL,
```

Comment:
We use `mediumblob` for our other error fields, we should do the same here. Also, please capitalize all keywords to follow SQL convention.